src/internal/types.ts

```diff
@@ -75,20 +77,19 @@ export const InternalAccountStructs: Record<
   [`${BtcAccountType.P2wpkh}`]: InternalBtcP2wpkhAccountStruct,
 };
 
-export const InternalAccountStruct = define(
-  'InternalAccount',
-  (value: unknown) => {
-    const account = mask(value, BaseKeyringAccountStruct);
+export const InternalAccountStruct = define<
+  InternalEthEoaAccount | InternalEthErc4337Account | InternalBtcP2wpkhAccount
```

Comment:
Added type here because when using define with typescript, the type becomes unknown when using typescript. https://docs.superstructjs.org/api-reference/types#custom-types